cogs/mod.py

```diff
@@ -204,7 +277,7 @@ def __init__(self, bot):
 
     @commands.command(no_pm=True, pass_context=True)
     @checks.admin_or_permissions(ban_members=True)
-    async def ban(self, ctx, user: discord.Member, days: int=0):
+    async def ban(self, ctx, user: discord.Member, days: int=0, *, reason: str = None):
```

Comment:
I can't find a good solution for this. As it is, the user would be forced to input the days to enter a reason and I really don't like it. I could parse it manually without relying on the command ext but dunno, it seems to get messy fast between defaults, sanity checks on the days, etc. Any suggestion?